current-eclipse-microprofile-sub-projects/config.adoc

```diff
@@ -160,9 +164,7 @@ can not.
 ====
 *New in MicroProfile Config 1.3*
 
-MicroProfile Config 1.3 introduces an mapping rule to the
-ConfigSource that provides the environment variables will then search for three
- environment variables from a single property name:
+MicroProfile Config 1.3 introduces	an	mapping	rule from config property name to environment variable. This rule	searches three	environment	variable variants for each	property	name:
```

Comment:
*a* mappring  rule.

@JStickler your rewording is better. For info, the feature in Config 1.3 is described at https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configsources.asciidoc#default-configsources